dart/lib/src/exception_cause.dart

```diff
@@ -0,0 +1,6 @@
+class ExceptionCause {
```

Comment:
Let's add docs for public classes and properties, that apply to some other files as well.